browser/src/control/Control.AddressInputField.ts

```diff
@@ -0,0 +1,134 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * JSDialog.AddressInputField - implementation of address input field in calc
+ */
+
+/* global JSDialog _ _UNO UNOKey app */
+
+class AddressInputField {
+	map: any;
+	parentContainer: HTMLElement;
+	builder: any;
+
+	public constructor(map: any) {
+		this.map = map;
+		this.parentContainer = L.DomUtil.get('addressInput');
+
+		this.map.on('jsdialogupdate', this.onJSUpdate, this);
+		this.map.on('jsdialogaction', this.onJSAction, this);
+		this.map.on('doclayerinit', this.onDocLayerInit, this);
+		this.map.on('celladdress', this.onCellAddress, this);
+
+		this.builder = new L.control.jsDialogBuilder({
+			mobileWizard: this,
+			map: this.map,
+			cssClass: 'addressInput jsdialog',
```

Comment:
please add shortcut value for WindowID here like we did for notebookbar: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.Notebookbar.js#L40

I commented also core part for that